JS/Lesson16/esra/script.js

```diff
@@ -173,52 +205,191 @@ isPositiveNegativeZero(0);
 // 25.0 – 29.9	Overweight
 // 30.0 and Above	Obesity
 
+function categorizeBMI(bmi) {
+	if (bmi < 18.5) {
+		console.log ("Underweight");
+	} else if (bmi >= 18.5 && bmi <= 24.9) {
+		console.log ("Normal");
+	} else if (bmi >= 25.0 && bmi <= 29.9) {
+		console.log ("Overweight");
+	} else {
+		console.log ("Obesity");
+	}
+  }
+  categorizeBMI(26.0);
+  categorizeBMI(15);
+
+
+
 // 9. Write a program that takes in a person's age and checks if they are a child (age 0-12), teenager (age 13-19), adult (age 20-64), or senior citizen (age 65 and above) using if else statement.
+function calculateages(age) {
+	if (age < 12) {
+		console.log ("Child");
+	} else if (age >= 13 && age <= 19) {
+		console.log ("teenager");
+	} else if (age >= 20 && age <= 64) {
+		console.log ("adult");
+	} else {
+		console.log ("senior citizen");
+	}
+  }
+
+  calculateages(16);
+
+
+
 
 // 10. Write a program that takes age as a number and whether they are a student (yes or no). If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". If the age is greater than 65, print "You are a senior citizen". If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
 
-// 11. Write a program that asks the user to enter their salary and their years of experience. If the salary is less than $30,000 and the years of experience are less than 5, print "Sorry, we cannot offer you the job at this time". If the salary is less than $50,000 and the years of experience are between 5 and 10 (inclusive), print "We can offer you the job at a lower salary". If the salary is greater than or equal to $50,000 or the years of experience are greater than or equal to 10, print "We can offer you the job at the requested salary".
+function checkVotingEligibility(age, isStudent) {
```

Comment:
In programming, when you have value that can only be 'yes' or 'no', it's the most common to use a boolean type for it. In this case, `isStudent` should be a boolean instead of a string.

Reply:
I read and tried but I don't understand, how can I do boolean?

Reply:
See comments below, that's how you'd use boolean here